src/utils/fileName/fileName.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { decryptWallet } from "../../common/config/decrypt";
+import sampleConfig from "../../test/fixtures/sample-config.json";
+import { Config, ConfigFile } from "../../types";
+import { generateFileName } from "./fileName";
+
+describe("generateFileName", () => {
+  it("should generate the file name correctly with the given config and file name", async () => {
+    const wallet = await decryptWallet(sampleConfig as ConfigFile, "password");
+    const config = { ...sampleConfig, wallet } as Config;
+    const fileName = generateFileName(config, "document-1", "tt");
+
+    expect(fileName).toStrictEqual("document-1-ropsten.tt");
+  });
+
+  it("should generate the file name correctly when config.network is 'homestead'", async () => {
+    const wallet = await decryptWallet(sampleConfig as ConfigFile, "password");
+    const config = { ...sampleConfig, network: "homestead", wallet } as Config;
+    const fileName = generateFileName(config, "document-1", "tt");
+
+    expect(fileName).toStrictEqual("document-1.tt");
+  });
+});
```

Comment:
nice, maybe 1 more test case for extension change + expectation

Reply:
ok